### PR TITLE
Russian Phone Numbers

### PR DESCRIPTION
**Pull request by SofaKing18**
Hello, added Russian phone number formats, which starting with +7 or 8
Home numbers with actual region codes

### CODE REVIEW

**Comment by AlexWayfer** [comment]
Good PR!

File: lib/ffaker/phone_number_ru.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+module FFaker
+  #
+  module PhoneNumberRU
+    extend ModuleUtils
+    extend self
+
+    COUNTRY_PREFIX = %w[+7 8].freeze
+    MOBILE_PREFIX = *(900..999).freeze
+    PHONE_NUMBER = ['#######'].freeze
+    HOME_REGION_PREFIX = %w["0 301 302 336 341 342 343 345 346 347 349 351 352
```
Comment (marocchino) [inline lib/ffaker/phone_number_ru.rb]: `%w["0` -> `%w[0`
`879 "].freeze` -> `879].freeze`

File: lib/ffaker.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module FFaker
-  VERSION = '2.2.0'.freeze
+  VERSION = '2.7.1'.freeze
```
Comment (marocchino) [inline lib/ffaker.rb]: please revert this.

File: Changelog.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+## 2.7.1
```
Comment (marocchino) [inline Changelog.md]: !`## master` will be better!

File: test/test_phone_number_ru.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+# encoding: utf-8
+
+require 'helper'
+
+# Author: wiseleyb<wiseleyb@gmail.com>
```
Comment (marocchino) [inline test/test_phone_number_ru.rb]: Is this needed? 💦

File: test/test_phone_number_ru.rb [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+class TestPhoneNumberRU < Test::Unit::TestCase
+  include DeterministicHelper
```
Comment (marocchino) [inline test/test_phone_number_ru.rb]: this module for deterministic test, something like:
```rb
assert_methods_are_deterministic(FFaker::PhoneNumberRU, :phone_number, :home_phone_number, :mobile_phone_number)
```

**Comment by Volosh1n** [comment]
Module already added in #397

**Pull request closed without merging**